python/cuml/cuml/tests/test_make_blobs.py

```diff
@@ -65,31 +66,36 @@ def test_make_blobs_scalar_parameters(
     shuffle,
     random_state,
     order,
+    device,
 ):
 
-    out, labels = cuml.make_blobs(
-        dtype=dtype,
-        n_samples=n_samples,
-        n_features=n_features,
-        centers=centers,
-        cluster_std=0.001,
-        center_box=center_box,
-        shuffle=shuffle,
-        random_state=random_state,
-        order=order,
-    )
-
-    assert out.shape == (n_samples, n_features), "out shape mismatch"
-    assert labels.shape == (n_samples,), "labels shape mismatch"
-
-    if order == "F":
-        assert out.flags["F_CONTIGUOUS"]
-    elif order == "C":
-        assert out.flags["C_CONTIGUOUS"]
-
-    if centers is None:
-        assert cp.unique(labels).shape == (3,), "unexpected number of clusters"
-    elif centers <= n_samples:
-        assert cp.unique(labels).shape == (
-            centers,
-        ), "unexpected number of clusters"
+    with using_device_type(device):
+
+        out, labels = cuml.make_blobs(
+            dtype=dtype,
+            n_samples=n_samples,
+            n_features=n_features,
+            centers=centers,
+            cluster_std=0.001,
+            center_box=center_box,
+            shuffle=shuffle,
+            random_state=random_state,
+            order=order,
+        )
+
+        assert out.shape == (n_samples, n_features), "out shape mismatch"
```

Comment:
Should we add an `assert` to check that `out` is a cupy/numpy array when the device type is gpu/cpu? At least that would be my expectation on how `make_blobs` should work